packages/preset-wind/src/variants/experimental.ts

```diff
@@ -0,0 +1,9 @@
+import type { Variant } from '@unocss/core'
+import { variantMatcher } from '@unocss/preset-mini/utils'
+
+export const variantStickyHover: Variant[] = [
```

Comment:
Can we add a `## Experimental` field in `preset-wind/README.md` describe it a bit? Thanks

Reply:
Do you think it is necessary to add warn on usage?

Reply:
Maybe. We could have a utility like `warnOnce` or something bound to the generator.